src/SmartComponents/DriftPage/DriftToolbar/__tests__/DriftToolbar.tests.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { Provider } from 'react-redux';
+import { shallow, mount } from 'enzyme';
```

Comment:
RTL should be used instead of enzyme.

Reply:
yes, but I didn't want to take time to convert this entire test file into rtl. that can be done later. I just did this to get a better snapshot temporarily.